### PR TITLE
Project - Mikaela

### PR DESCRIPTION
**Pull request by mikadsr**
Project - Mikaela De Smedt

### CODE REVIEW

File: _projects/2023/100481838/100481838.Rmd [hunk 1]
```diff
@@ -0,0 +1,1067 @@
+---
+title: "We feel it already. How about we visualize it?"
+description: |
+  A short description of the post.
```
Comment (Enchufa2) [inline _projects/2023/100481838/100481838.Rmd]: Please put a description here, a kind of abstract to introduce the post.

File: _projects/2023/100481838/100481838.Rmd [hunk 2]
```diff
@@ -0,0 +1,1067 @@
...
+categories: "2023"
+author: Mikaela De Smedt
+date: 2023-12-16
```
Comment (Enchufa2) [inline _projects/2023/100481838/100481838.Rmd]: Please use ```date: "`r Sys.Date()`"``` instead. The date will be automatically generated when the document is knitted.

File: _projects/2023/100481838/100481838.Rmd [hunk 3]
```diff
@@ -0,0 +1,1067 @@
...
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+# Introduction
```
Comment (Enchufa2) [inline _projects/2023/100481838/100481838.Rmd]: Use level two headings `##` as the first level. Level one corresponds to the project title that is generated from the header of the file.

File: _projects/2023/100481838/100481838.Rmd [hunk 4]
```diff
@@ -0,0 +1,1067 @@
...
+
+The aim of this tutorial is to analyze the visual encoding and replicate this chart. The final section also proposes additional improvements.
+
+## The plot in question: the story and how it's told
+
+The following chart uses data from retired NASA climate scientist James Hansen to show how summer temperatures have shifted toward more extreme heat over the past several decades.
+
+This representation uses bell curves to compare the distribution in the frequency of occurrence of temperature anomalies (y-axis) divided by local standard deviation for the 1951-1980 period (x-axis) obtained by binning all local results for the indicated region and 11-year period into 0.05 frequency intervals. Additionally, this chart is presented along an animation which shows the transition of all recorded periods.
+
+![Note: This is a screen recording of the actual animation. I was unable to properly download it due to the way it was embedded in the NYT website. Apologies for the resolution.](Untitled%20video%20-%20Made%20with%20Clipchamp.gif)
```
Comment (Enchufa2) [inline _projects/2023/100481838/100481838.Rmd]: This is nice for the presentation, but here the reader can go, click the link and see for themselves. Plus this video is too big. Please remove both the video and this reference to save some space in the repo. :)

File: _projects/2023/100481838/100481838.Rmd [hunk 5]
```diff
@@ -0,0 +1,1067 @@
...
+           fontface = 2)+
+  annotate("text", 
+           x=-5, 
+           y=-0.02, 
+           label= "Extremely Cold", 
+           color = "#085b84",
+           fontface = 2)+
+  annotate("text", 
+           x=-1.8, 
+           y=-0.02,
```
Comment (Enchufa2) [inline _projects/2023/100481838/100481838.Rmd]: I see that there is a lot of repeated code throughout the post, which makes the whole thing harder to read. To avoid this, you could encapsulate those things in reusable functions, something like:

```r
custom_annotations <- function(plot) {
  plot + annotate() + annotate() + annotate()
}
```

and then reuse it.

File: _projects/2023/100481838/100481838.Rmd [hunk 6]
```diff
@@ -0,0 +1,1067 @@
...
+
+```{r Import raw data, echo=TRUE}
+library(ggplot2)
+library(tidyverse)
+ccdata<- tibble(read.table(
+  file = "http://www.columbia.edu/~mhs119/PerceptionsAndDice/NH.JJA.11yrs.txt", 
+  header = TRUE,
+  skip = 1))
+
+head(ccdata)
```
Comment (Enchufa2) [inline _projects/2023/100481838/100481838.Rmd]: Please save the file as CSV, and read from the saved CSV instead of the remote URL, just in case it is not available in the future there. Also please remove the RData file.

File: _projects/2023/100481838/mysuperimportantfile.qmd [hunk 7]
```diff
@@ -0,0 +1,434 @@
+---
```
Comment (Enchufa2) [inline _projects/2023/100481838/mysuperimportantfile.qmd]: Is this file needed? Please remove.

**Review by Enchufa2 — CHANGES_REQUESTED** [review]
Nice work! I left you some comments with some fixes and improvements. As always, just modify, re-knit, add, commit and push. And please let me know with a comment when the changes are ready for a new review.

**Comment by Enchufa2** [comment]
BTW, converting back to draft, because it's easier for me to look for green/gray in the PR list. Please click "ready for review" again when you want me to make another pass.

**Review by Enchufa2 — APPROVED** [review]
Superb, congrats! Merging with minor fixes.

**Comment by Enchufa2** [comment]
And here's your work: https://csslab.uc3m.es/dataviz/projects/2023/100481838/

**Comment by mikadsr** [comment]
Amazing! Thank you so much for the course! Doing the blogpost was super fun.
